api/v1beta1/network_types.go

```diff
@@ -174,6 +174,20 @@ type VPCSpec struct {
 	// Defaults to 10.0.0.0/16.
 	CidrBlock string `json:"cidrBlock,omitempty"`
 
+	// EnableIPv6 requests an Amazon-provided IPv6 CIDR block with a /56 prefix length for
+	// the VPC. You cannot specify the range of IP addresses, or the size of the
+	// CIDR block.
+	// +optional
+	EnableIPv6 bool `json:"enableIPv6"`
+
+	// IPv6CidrBlock is the CIDR block provided by Amazon when VPC has enabled IPv6.
+	// +optional
+	IPv6CidrBlock string `json:"ipv6CidrBlock,omitempty"`
+
+	// IPv6Pool is the IP pool which must be defined in case of BYO IP is defined.
+	// +optional
+	IPv6Pool string `json:"ipv6Pool,omitempty"`
```

Comment:
I'm not using this one yet, but will once BYO is enabled... It is required for bring your own cidr for IPv6.